lib/eslint.js

```diff
@@ -166,6 +168,14 @@ function addDeclaredGlobals(program, globalScope, config) {
         }
         variable.writeable = explicitGlobals[name];
     });
+
+    // mark all exported variables as such
+    Object.keys(exportedGlobals).forEach(function(name) {
+        var variable = getVariable(globalScope, name);
+        if (variable) {
+            variable.eslintExported = true;
```

Comment:
The way it's currently being used I can probably use this just fine. It doesn't apply it to globals.